x-pack/test/api_integration/apis/infra/index.js

```diff
@@ -5,6 +5,7 @@
  */
 
 
+console.log("Did you load?");
```

Comment:
I guess this is a left-over `console.log`?

Reply:
oopsie!